contrib/fts-server/.osg_build.json

```
@@ -0,0 +1,3 @@
+{
+  "uses": "./.github/workflows/build-and-push--fts-server.yml"
```

```suggestion
  "uses": "./contrib/fts-server/.github/workflows/build-and-push--fts-server.yml"
```

I'd prefer that image specific workflows live in their own dirs